### PR TITLE
Add Workflow.set_tasks_from_layers()

### PR DESCRIPTION
**Pull request by dylanholmes**
Changes:
- Add `Workflow.set_tasks_from_layers()`, see unit tests for example usage.

### CODE REVIEW

File: griptape/structures/workflow.py [hunk 1]
```diff
@@ -89,6 +89,35 @@ def insert_task(
 
         return task
 
+    def set_tasks_from_layers(self, layers: list[BaseTask | list[BaseTask]]) -> None:
```
Comment (collindutter) [inline griptape/structures/workflow.py:92]: Can we just override `add_tasks` instead of introducing a new method?

**Review by collindutter — COMMENTED** [review]
Can you please update the docs to use whatever syntax we land on here?

**Comment by collindutter** [comment]
Also, what happened to your syntax proposed offline? I think I like the flexibility of something like that over this that only supports a relatively specific set of topologies.

**Comment by dylanholmes** [comment]
Discarding this in favor of a PR that modifies the api for BaseTask.

**Pull request closed without merging**